Resources/Prototypes/Entities/Objects/Misc/land_mine.yml

```diff
@@ -68,9 +82,25 @@
   id: LandMineExplosive
   components:
   - type: ExplodeOnTrigger
+  - type: LandMine
```

Comment:
you don't need to add it again, because the parent has it

Reply:
Fixed